aQute.libg/src/aQute/libg/reporter/ReporterMessages.java

```diff
@@ -84,7 +84,7 @@ public Object invoke(Object target, Method method, Object[] args) throws Throwab
 						char c = name.charAt(i);
 						switch (c) {
 							case '_' :
-								sb.append(" %s, ");
```

Comment:
I am a bit worried that this change could invalidate users' `-fixupmessages` instructions.